### PR TITLE
(DOCSP-335): Update with Windows install.

### PR DESCRIPTION
**Pull request by atsansone**
@i80and : Can you see if this looks correct? I did base it on my work with Ops Manager installs.

### CODE REVIEW

File: source/includes/steps-install-bi-connector.yaml [hunk 1]
```diff
@@ -41,14 +60,31 @@ pre: |
...
+  - heading: "Windows"
+    pre: |
+
+      a. Extract the ZIP file you downloaded.
+      b. Save the |bi| directory into a directory you listed in your system ``PATH``.
+
+         .. cssclass:: copyable-code
+         .. code-block:: powershell
+
+            move * "C:\Program Files\MongoDB BI Connector"
```
Comment (i80and) [inline source/includes/steps-install-bi-connector.yaml]: For consistency with the Unix instructions, should this be `move bin/* ...`?

**Comment by atsansone** [comment]
@i80and : Please merge.

**Comment by i80and** [comment]
Right, sorry. I'm still getting used to this

**Comment by i80and** [comment]
...and deployed!

**Pull request merged**